src/james.data.clss.pas

```diff
@@ -36,16 +36,16 @@   TDataStream = class sealed(TInterfacedObject, IDataStream)
   private
     FStream: TMemoryStream;
   public
-    constructor Create(Stream: TStream);
-    constructor Create(const S: string);
-    class function New(Stream: TStream): IDataStream;
-    class function New(const S: string): IDataStream;
-    class function New(Strings: TStrings): IDataStream;
+    constructor Create(Stream: TStream); Overload;
```

Comment:
@nunopicado every keyword should be in lowercase, so `overload` instead of `Overload`.